### ISSUE TITLE
[Term Entry] Dart List .add()

### ISSUE THREAD

**Issue by christian-dinh** [Codecademy/docs#4299]
### Reason/inspiration (optional)

We would like a new term entry under Dart [List](https://www.codecademy.com/resources/docs/dart/concept/list) for the Dart List `.add()` method. 
Provide the entry in a markdown file under `docs/content/dart/concept/list/terms/add/add.md`

The entry should
    - Have an introductory paragraph about what the method is and does.
    - A syntax section with a pseudo code block with an explanation of it.
    - An example showing the method in use.

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by AnisurRahmanAR** [comment]
Hi there, I would like to take on this issue please. I've found this issue through the CodeAcademy "Git and GitHub" modules, as part of my "Contributing to the CodeAcademy Docs Repository" project. Just for further clarity, the request is that the Dart term .add() needs to be added to a directory I need to create - list/terms/add/ where the markdown file will be located?

**Comment by AnisurRahmanAR** [comment]
Hi again, checking in to ask if it would be possible to take on this issue please?

**Comment by KTom101** [comment]
@AnisurRahmanAR You're assigned. You are correct you will need to create `list/terms/add/`.

In addition to the documents linked in the description, please also look at our [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) if you haven't already.  After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

**Comment by AnisurRahmanAR** [comment]
Hi KTom101, I believe I've created the file and followed each of the guides, and I've even pushed to changes to get reviews but I can't make the pull request for some reason. 
![Screenshot 2024-02-18 035554](https://github.com/Codecademy/docs/assets/157653527/f82698bb-271b-42ad-b5f1-77ca17f7dc48)
I keep coming into this image despite having the option to compare and pull request. 
![Screenshot 2024-02-18 035728](https://github.com/Codecademy/docs/assets/157653527/5d6bce00-8467-4e4b-a64b-937757cbdc9d)
I'm not entirely sure what's causing this, and I have tried multiple times to start from the beginning and also searching up solutions. As a beginner, I don't know the work around for this. Sorry for the inconvenience.
